Sources/Rendering/Core/InteractorObserver/index.d.ts

```diff
@@ -0,0 +1,132 @@
+import vtkRenderWindowInteractor from '../RenderWindowInteractor';
+import vtkRenderer from '../../../Rendering/Core/Renderer';
+import { vtkObject, EventHandler, vtkSubscription } from '../../../interfaces';
+import { Vector3 } from '../../../types';
+
+export interface vtkInteractorObserver extends vtkObject {
+  /**
+   * Invoke an interaction event.
+   * 
+   * @param args Event payload.
+   */
+  invokeInteractionEvent(...args: unknown[]): void;
```

Comment:
i am not sure about the usage of the spread operator here

Reply:
`invoke*` can take any number of unknown args, and will forward them on to the event listeners. Though in practice, I haven't seen more than 1 argument to any of the invoke calls.

Reply:
I think we can leave it as-si